Calibration/HcalCalibAlgos/test/python/isoTrackRecoAnalysis_cfg.py

```diff
@@ -26,7 +26,7 @@
 process.towerMakerAll.ecalInputs = cms.VInputTag(cms.InputTag("ecalRecHit","EcalRecHitsEB"), cms.InputTag("ecalRecHit","EcalRecHitsEE"))
 process.towerMakerAll.AllowMissingInputs = True
 
-process.load('Calibration.HcalCalibAlgos.HcalIsoTrkAnalyzer_cff')
+process.load('Calibration.HcalCalibAlgos.hcalIsoTrkAnalyzer_cff')
```

Comment:
@bsunanda sorry for the maybe trivial question, but why did you change `HcalIsoTrkAnalyzer_cff` to `hcalIsoTrkAnalyzer_cff` (and same in the other .py files in this PR)? 
In the repo i still see the file with the capital letter:
https://github.com/cms-sw/cmssw/blob/master/Calibration/HcalCalibAlgos/python/HcalIsoTrkAnalyzer_cff.py

Reply:
Sorry i just realized that it is renamed in this PR! In that case that's fine!